docs/components/table/README.md

```diff
@@ -382,12 +383,28 @@ will emit the `filtered` event, passing a single argument which is the complete
 items passing the filter routine. Treat this argument as read-only.
 
 ### Sorting
+As mentioned above in the **fields** section, you can make columns sortable. Clciking on
+sortable a column header will sort the column in ascending direction, while clicking
+on it again will switch the direction or sorting.  Clicking on a non-sortable column
```

Comment:
OK good... The docs example should show the sort key and order under the table, using the `.sync` modifier.